test/evergreen.yml

```diff
@@ -244,6 +245,25 @@ functions:
             # Save generated documentation
             mv docs docs-$branch
           done
+  "dump stacktraces": &dump_stacktraces
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/cmake_build"
```

Comment:
The `cmake_build` directory won't be created for Windows (and this function will fail silently as part of the post-task). It's fine if this change is mainly targeted for Linux tasks.

Reply:
I take it back. It looks like the Windows compile task creates the same `cmake_build` directory.